metaseq/hub_utils.py

```diff
@@ -435,38 +435,6 @@ def _build_batches(
         return batch_iterator
 
 
-class BPEHubInterface(object):
```

Comment:
Should we keep these tests until we remove the items they're testing?

Reply:
Reverted the deletion